packages/hub/src/graphql/mutations/updateModelPrivacy.ts

```diff
@@ -0,0 +1,40 @@
+import { prisma } from "@/prisma";
+import { builder } from "@/graphql/builder";
+
+import { Model } from "../types/Model";
+
+builder.mutationField("updateModelPrivacy", (t) =>
```

Comment:
Just fyi, this function looks pretty long to me for what it does. I imagine there might be ways to DRY up this sort of thing later.

Reply:
Yes, and it's copy-pasted from `updateModelSlug`, but I'm not sure yet which parts should be reused. Will refactor on the next change or when the next update endpoint becomes necessary.

(Or maybe we should have a single `updateModel` instead with multiple nullable args.)